packages/js-dapi-client/polyfills/fetch-polyfill.js

```diff
@@ -2,7 +2,7 @@
 
 const { default: fetch, Headers, Request, Response } = require('node-fetch');
```

Comment:
Is it polyfill for browser?

Reply:
No, it is polyfill for Node.js, because fetch already bundled in both environments, but in Node environment it is based on undici, that is hard to bundle in the web yet., so it is being replace with node fetch

Reply:
So we replace built-in `fetch` to `node-fetch`? What if we just do not do that?

Reply:
Js-dapi-client will be unable to connect with self-signed SSL hosts, because the code accepting such certificates is built on top of `http` library's agent, while built-in fetch is using `undici`. We can enable undici in the project and completely remove node-fetch, but its not yet ready for bundling in the web.

Reply:
Yes you can, there is a webpack config for it already.

There is another approach tho, by using `cross-fetch` instead of `node-fetch`, but it uses factory approach to instantiate fetch, and a little code refactor will be required

Reply:
So undici doesn't accept http user agent?

Reply:
Discussed in Slack. It's only JSON RPC that is temporary thing and will be replaced with gRPC before MVP